jetty-core/jetty-start/src/main/java/org/eclipse/jetty/start/BaseBuilder.java

```diff
@@ -319,14 +319,17 @@ else if (module.isTransitive())
                 {
                     if (module.hasIniTemplate())
                     {
-                        StartLog.info("%-15s transitively enabled, ini template available with --add-modules=%s",
-                            module.getName(),
-                            module.getName());
-                    }
-                    else
-                    {
-                        StartLog.info("%-15s transitively enabled", module.getName());
+                        try
+                        {
+                            builder.get().addModule(module, startArgs.getJettyEnvironment().getProperties());
+                        }
+                        catch (IOException e)
+                        {
+                            throw new RuntimeException(e);
+                        }
```

Comment:
There already is a `try/catch` few lines above, that performs the same `builder.get().addModule(...)`, so I would move all the code inside that existing `try/catch`.

Actually, since the logic is very similar, can it be merged?

Reply:
Solved with alternate solution.